include/openssl/rand.h

```diff
@@ -82,6 +82,9 @@ OPENSSL_EXPORT void RAND_seed(const void *buf, int num);
 // RAND_load_file returns a nonnegative number.
 OPENSSL_EXPORT int RAND_load_file(const char *path, long num);
 
+// RAND_write_file does nothing and returns negative 1
```

Comment:
```suggestion
// RAND_write_file does nothing and returns negative 1.
```

Inconsistent use of `.`s :)